.circleci/config.yml

```diff
@@ -150,7 +150,7 @@ commands:
         default: false
       env:
         type: enum
-        enum: [ staging, alpha ]
+        enum: [staging, alpha]
```

Comment:
I guess this was eslint?

Reply:
probably, I think it does md too, but perhaps this wasn't .eslintignored

Reply:
Prettier formats several file types, including YAML.
https://prettier.io/docs/en/index.html